lib/pigeon.ex

```diff
@@ -13,7 +13,11 @@ defmodule Pigeon do
   end
 
   defp workers do
-    adm_worker() ++ apns_workers()
+    adm_worker() ++ apns_workers() ++ task_supervisors
```

Comment:
Nitpick: `task_supervisors()` with parentheses to avoid Elixir 1.4 compiler warning